pkgs/by-name/fl/flow-editor/build.zig.zon.nix

```diff
@@ -0,0 +1,195 @@
+# generated by zon2nix (https://github.com/Cloudef/zig2nix)
+
+{
+  lib,
+  linkFarm,
+  fetchurl,
+  fetchgit,
+  runCommandLocal,
+  zig,
+  name ? "zig-packages",
+}:
+
+with builtins;
+with lib;
```

Comment:
Can you please avoid this at the top of the file? Import only what you need.

If this file is automatically generated, please ignore the comment.

Reply:
#autoFacepalm